test/models/observation_test.rb

```diff
@@ -88,6 +87,37 @@ def test_name_been_proposed
     assert(!observations(:coprinus_comatus_obs).name_been_proposed?(names(:conocybe_filaris)))
   end
 
+  def test_vote_counts
+    obs = observations(:owner_only_favorite_eq_consensus)
+    assert_equal(0, obs.users_votes(users(:rolf)).count)
```

Comment:
It's also not clear to me.  As best I can reconstruct events, I wrote this to test either (a) my changes to fixtures, and/or (b) a change to `users_votes` which I was contemplating, but never implemented.  See [MO Developers › Loop vs Finder](https://groups.google.com/forum/#!topic/mo-developers/uBfhRMGbV3k).
So I will delete the test.